services/user.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None) -> None:
+    new_user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+    if email:
+        new_user.email = email
+    if first_name:
+        new_user.first_name = first_name
+    if last_name:
+        new_user.last_name = last_name
+    new_user.save()
+
+
+def get_user(user_id: int) -> get_user_model().objects:
```

Comment:
same, type annotation, u can use User model for type annotation

Reply:
Fixed